README.md

```diff
@@ -97,66 +103,55 @@ To get this application up and running you will need the following tools:
+Routemaster needs to have a RabbitMQ virtual host to connect to.  By default
+this is going to be called `routemaster.development`.
 
 - Check if RabbitMQ is running by pointing your browser to [http://localhost:15672/#/](http://localhost:15672/#/)
 - Login with guest/guest
 - Go to _Admin => Virtual Hosts_ and add a _New virtual host_ named `routemaster.development`
 - Click on the newly added virtual host and add the guest user with the default permissions
 
-Routemaster only accepts HTTPS calls.
-To get around this restriction on development we can create a tunnel such that
-the requests to our HTTPS port goes to the normal HTTP port.
-You can use the **tunnels** gem to do that.
-
-```
-gem install tunnels
-sudo tunnels 127.0.0.1:443 127.0.0.1:80
-```
-
-This command creates a tunnel between port 443 (the default SSL port) and your 80 port.
+Alternatively:
 
-This is not enough since you need to forward the calls arriving at port 80 to the actual routemaster port, the default is 17890.
+```bash
+rabbitmqctl add_vhost routemaster.development
+rabbitmqctl set_permissions -p routemaster.development guest ".*" ".*" ".*"
 
-We can use [Pow](http://pow.cx/) to do that.
+```
 
-- Install it here https://github.com/basecamp/pow
-- Configure Port Proxying to to forward requests arriving at http://localhost:80 to http://routemaster.dev:17890.
+Routemaster only accepts HTTPS calls. To get around this restriction on
+development, please install [`puma-dev`](https://github.com/puma/puma-dev).
 
-This last step is as simple as creating a file with a port number in the .pow folder
+Then proxy routemaster requests by running the following:
 
 ```
-$ echo 17890 > ~/.pow/routemaster
+$ echo 17890 > ~/.puma-dev/routemaster
 ```
 
 Now all your calls to `https://routemaster.dev` should correctly arrive at `http://127.0.0.1:17890`.
 
-You will probably need Routemaster to contact your app on HTTPS to deliver events.
-To do that just repeat the POW step to add a Port Proxying to your app.
-
-`$ echo <your-app-port> > ~/.pow/<your-app-name>`
+You will also need Routemaster to contact your app through HTTPS to deliver
+events.  Follow the same steps above to proxy your app requests, i.e. for a
+Rails app that would be
 
-which for rails will probably be
+`$ echo 3000 > ~/.puma-dev/myapp`
 
-`$ echo 3000 > ~/.pow/<your-app-name>`
-
-You can register your app to Routemaster and provide as a callback url for events
-`https://<your-app-name>.dev/<your-app-route>`
-
-
-#### Running it
+Running it
+---
 
 To run the Routemaster application locally you can use the **foreman** tool:
+
 ```
 foreman start
 ```
 This will start both the **web** and **watch** processes. Keep in mind that the
 default web port that the **web** process will listen to is defined in the .env
-file.
+file. By default routemaster log level is set to `DEBUG` if this is too chatty
+you can easily configure this in the `.env` file
 
 
-### Scaling Routemaster out
+Scaling Routemaster out
```

Comment:
Just found it very hard to read on Github's rendition because some titles are rendered very small, similar to **bold** as opposed to an actual tittle. 

I suppose I can revert that change since now we end up having two styles.